### PR TITLE
OPENNLP-1153: add model download page

### PR DESCRIPTION
**Pull request by wcolen**

### CODE REVIEW

**Comment by kojisekig** [comment]
+1 lgtm

**Comment by kinow** [comment]
+1

File: src/main/jbake/content/models.ad [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+|===
+| Component | Language | Compatibility | Description | README and Reports | File | Signatures
+
+| Language Detector
+| Multilanguage
+| >= 1.8.3
+| Detects 103 languages in ISO 693-3 standard. Works well with longer texts that have at least 2 sentences or more from the same language.
+| https://www.apache.org/dist/opennlp/models/README[README]
+https://www.apache.org/dist/opennlp/models/langdetect-183.bin.report.txt[Effectiveness]
+https://www.apache.org/dist/opennlp/models/langdetect-183.bin.report.txt[Misclassified]
```
Comment (kottmann) [inline src/main/jbake/content/models.ad]: This seems wrong, the file is the same as the one before.

**Comment by kottmann** [comment]
We should also rephrase the download page:
http://opennlp.apache.org/download.html

There it says models can be found at SourceForge, but now it should point to our model page.

File: src/main/jbake/content/models.ad [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+Use the links in the table below to download the pre-trained models for the Apache OpenNLP.
+
+Note: All models are zip compressed (like a jar file), they *must not* be uncompressed.
+
+[options="header"]
+|===
+| Component | Language | Compatibility | Description | README and Reports | File | Signatures
+
+| Language Detector
+| Multilanguage
```
Comment (kottmann) [inline src/main/jbake/content/models.ad]: We should give this model a name.

File: src/main/jbake/content/models.ad [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+| https://www.apache.org/dist/opennlp/models/README[README]
+https://www.apache.org/dist/opennlp/models/langdetect-183.bin.report.txt[Effectiveness]
+https://www.apache.org/dist/opennlp/models/langdetect-183.bin.report.txt[Misclassified]
+|https://www.apache.org/dyn/closer.cgi/opennlp/models/langdetect-183.bin[langdetect-183.bin]
+|https://www.apache.org/dist/opennlp/models/langdetect-183.bin.md5[md5]
+https://www.apache.org/dist/opennlp/models/langdetect-183.bin.sha1[sha1]
+https://www.apache.org/dist/opennlp/models/langdetect-183.bin.asc[asc]
+|===
+
+# Older Models
```
Comment (kottmann) [inline src/main/jbake/content/models.ad]: This I would just call SourceForge models, we always refer to them like that.

File: src/main/jbake/content/download.ad [hunk 4]
```diff
@@ -75,8 +75,8 @@ More information about release signing and verifying signatures can
 be found https://www.apache.org/dev/release-signing.html[here].
 
 ## Models
-The models over at SourceForge for 1.5.0 can be http://opennlp.sourceforge.net/models-1.5/[found here]
-and are fully compatible with Apache OpenNLP {opennlp_version}.
+
+The models for Apache OpenNLP can be link:/models.html[found here].
```
Comment (smarthi) [inline src/main/jbake/content/download.ad]: 'are' found here - not 'can be'

**Review by kottmann — APPROVED** [review]
+1 lgtm

**Pull request merged**